notification/tests.py

```diff
@@ -15,7 +15,29 @@ def test_delete_notification_from_database(self, notification):
         with pytest.raises(Notification.DoesNotExist):
             Notification.objects.get(pk=notification.pk)
 
-    def test_mark_as_read_notification(self, notification):
+    def test_update_notification_message(self, notification):
+        new_notification = notification
+        assert notification.message == 'Test notification'
+        notification.message = 'ABCDE'
+        notification.save()
+        updated_notification = Notification.objects.get(pk=new_notification.pk)
+        assert updated_notification.message == 'ABCDE'
+        assert new_notification.id == notification.id == updated_notification.id
+
+    def test_update_notification_type(self, notification):
+        new_notification = notification
```

Comment:
Same here

Reply:
Please check the comment above :-)

Reply:
Hey! I updated this code, please take a look and leave any comment :-) 
I resolve this converstaion for now as it's irrelevant after the update.